src/components/Records/Shared/AbbreviationTooltip.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-tooltip
+    v-if="abbreviation!==null && abbreviation.length > 40"
```

Comment:
Do you have an example that I can use to showcase this feature please?

Reply:
yes everytime there is an abbreviation available so its not null and its length its bigger than 40 then we ll show the tooltip. see the below picture. 

![image](https://user-images.githubusercontent.com/25683981/104935817-9e946a00-59a3-11eb-9912-2f65512a1b61.png)